src/pybind/mgr/dashboard/controllers/rgw_iam.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+from ..security import Scope
+from ..services.rgw_iam import RgwAccounts
+from ..tools import str_to_bool
+from . import APIDoc, APIRouter, EndpointDoc, RESTController, allow_empty_body
+
+
+@APIRouter('rgw/accounts', Scope.RGW)
+@APIDoc("RGW User Accounts API", "RgwUserAccounts")
+class RgwUserAccountsController(RESTController):
+
+    @allow_empty_body
+    def create(self, account_name: Optional[str] = None,
+               account_id: Optional[str] = None, email: Optional[str] = None):
+        accounts = RgwAccounts()
+        return accounts.create_account(account_name, account_id, email)
```

Comment:
```suggestion
    def create(self, account_name: Optional[str] = None,
               account_id: Optional[str] = None, email: Optional[str] = None):
        return RgwAccounts.create_account(account_name, account_id, email)
```

Reply:
ohh yess, I forgot to make this change
Thanks will update my PR.

Reply:
Fixed!